### PR TITLE
Attempt to fix probo tests

### PR DESCRIPTION
**Pull request by mrkmiller**

### CODE REVIEW

**Comment by mrkmiller** [comment]
This passes all tests now except for the known article rss feed.

File: .probo.yaml [hunk 1]
```diff
@@ -38,10 +38,13 @@ steps:
   - name: Run Behat JS
     plugin: Script
     script:
-      - apt-get update > /dev/null 2>&1
-      - apt-get install google-chrome-stable xvfb -y > /dev/null 2>&1
-      - wget http://chromedriver.storage.googleapis.com/2.40/chromedriver_linux64.zip > /dev/null 2>&1
-      - unzip chromedriver_linux64.zip -d /usr/bin
+      - wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | sudo apt-key add -
```
Comment (dearmond) [inline .probo.yaml:41]: You shouldn't have to add this. apt-get has Google Chrome

Comment (dearmond) [reply .probo.yaml:41]: Maybe the "sudo" in line 47 is the key.

**Comment by mrkmiller** [comment]
I think something is up with Probo. This no longer works. Also, now the ssh key is failing. In addition, a different pull request passed js tests when it should have failed all of them.

**Comment by mrkmiller** [comment]
this has been fixed elsewhere.

**Pull request closed without merging**